perftests/scheduler_simulator/test_alg.go

```diff
@@ -427,8 +427,8 @@ func (st *SchedulingAlgTester) makeTestCluster(num int) *testCluster {
 	return h
 }
 
-func (st *SchedulingAlgTester) getTestConfig() server.SchedulerConfig {
-	return server.SchedulerConfig{
+func (st *SchedulingAlgTester) getTestConfig() server.SchedulerConfiguration {
```

Comment:
jw - why the rename to SchedulerConfiguration? Makes this diff bigger/harder to pick out which parts are actual changes

Reply:
I (and json marshaling) was tripping over multiple uses of SchedulerConfig - 'SchedulerConfig' in the json structure is being parsed to the SchedulerJSONConfig field, then the code translates that to the scheduler's SchedulerConfiguration object.

Reply:
oof gotcha